src/mesh/domain.jl

```diff
@@ -222,9 +222,14 @@ LazyOperators.pretty_name(domain::BoundaryFaceDomain) = "BoundaryFaceDomain"
 
 indices(d::BoundaryFaceDomain) = get_cache(d)
 
-function BoundaryFaceDomain(mesh::Mesh, bc::PeriodicBCType)
-    cache =
-        _compute_periodicity(mesh, labels_master(bc), labels_slave(bc), transformation(bc))
+function BoundaryFaceDomain(mesh::Mesh, bc::PeriodicBCType; tol = 1e-9)
```

Comment:
I think it would be better to pass `tol` to the constructor of `PeriodicBCType`

Reply:
I agree, I'll do it when I can